atlas-db-scxa-base/image_tag

```diff
@@ -0,0 +1 @@
+atlas-db-scxa-base:1.0
```

Comment:
Other tags in the repo are on the format 0.0.1
should we follow the same convention?